running-a-nats-service/configuration/clustering/cluster_config.md

```diff
@@ -10,7 +10,7 @@ The [`cluster` configuration map](../../configuration/README.md#clustering) has
 | `advertise` or `cluster_advertise` | Hostport `<host>:<port>` to advertise how this server can be contacted by other cluster members. This is useful in setups with NAT. When using TLS this is important to set to control the hostname that clients will use when discovering the route since by default this will be an IP, otherwise TLS hostname verification may fail with an IP SANs error.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-| `no_advertise`                     | When set to 'true', the server will not send or gossip its client URLs to other servers in the cluster and will not tell its client about the other servers' client URLs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `no_advertise`                     | When set to 'true', the server will not gossip its server URLs to clients .                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Would it make sense to specify that "clients" here mean "other NATS server instances connecting directly to it"?